src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -63,6 +63,51 @@ public void Kill()
             }
         }
 
+        /// <summary>Stops the associated process immediately.</summary>
+        private void Stop()
+        {
+            EnsureState(State.HaveNonExitedId);
+            if (Interop.Sys.Kill(_processId, Interop.Sys.Signals.SIGSTOP) != 0)
+            {
+                throw new Win32Exception(); // to match behavior of Kill()
```

Comment:
Currently, don't have a test verifying that this exception is raised. Any ideas on how force this throw to happen in a test?

Reply:
I'd try killing the process with syscall perhaps so that you got invalid PID when you kill it, if doesn't work then well... I wouldn't spend too much time on this

Reply:
Thanks!